default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	//  - cp950: Traditional Chinese (Big5)
+	//  - big5hkscs: Traditional Chinese (Big5-HKSCS)
+	//  - shiftjis: Japanese (Shift JIS)
+	//  - eucjp: Japanese (EUC-JP)
+	//  - euckr: Korean (EUC-KR)
+	//  - windows874: Thai (Windows 874)
+	//  - iso885911: Latin/Thai (ISO 8859-11)
+	//  - koi8ru: Cyrillic (KOI8-RU)
+	//  - koi8t: Tajik (KOI8-T)
+	//  - gb2312: Simplified Chinese (GB 2312)
+	//  - cp865: Nordic DOS (CP 865)
+	//  - cp850: Western European DOS (CP 850)
+	"files.encoding": "utf8",
+
+	// The default end of line character.
+	//  - \n: LF
+	//  - \r\n: CRLF
+	//  - auto: Uses operating system specific end of line character.
+	"files.eol": "auto",
+
+	// Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for excluding files and folders. For example, the file explorer decides which files and folders to show or hide based on this setting. Refer to the `search.exclude` setting to define search-specific excludes.
+	"files.exclude": {
+		"**/.git": true,
+		"**/.svn": true,
+		"**/.hg": true,
+		"**/CVS": true,
+		"**/.DS_Store": true,
+		"**/Thumbs.db": true,
+		"**/*.crswap": true
+	},
+
+	// Controls whether unsaved files are remembered between sessions, allowing the save prompt when exiting the editor to be skipped.
+	//  - off: Disable hot exit. A prompt will show when attempting to close a window with editors that have unsaved changes.
+	//  - onExitAndWindowClose: Hot exit will be triggered when the browser quits or the window or tab is closed.
+	"files.hotExit": "onExitAndWindowClose",
+
+	// When enabled, insert a final new line at the end of the file when saving it.
+	"files.insertFinalNewline": false,
+
+	// Timeout in milliseconds after which file participants for create, rename, and delete are cancelled. Use `0` to disable participants.
+	"files.participants.timeout": 60000,
+
+	// Restore the undo stack when a file is reopened.
+	"files.restoreUndoStack": true,
+
+	// A save conflict can occur when a file is saved to disk that was changed by another program in the meantime. To prevent data loss, the user is asked to compare the changes in the editor with the version on disk. This setting should only be changed if you frequently encounter save conflict errors and may result in data loss if used without caution.
+	//  - askUser: Will refuse to save and ask for resolving the save conflict manually.
+	//  - overwriteFileOnDisk: Will resolve the save conflict by overwriting the file on disk with the changes in the editor.
+	"files.saveConflictResolution": "askUser",
+
+	// Enables the simple file dialog. The simple file dialog replaces the system file dialog when enabled.
+	"files.simpleDialog.enable": false,
+
+	// When enabled, will trim all new lines after the final new line at the end of the file when saving it.
+	"files.trimFinalNewlines": false,
+
+	// When enabled, will trim trailing whitespace when saving a file.
+	"files.trimTrailingWhitespace": false,
```

Comment:
true

Reply:
This is _also_ a built in setting? Nice.